sklearn/metrics/pairwise.py

```diff
@@ -1639,7 +1647,7 @@ def _pairwise_callable(X, Y, metric, force_all_finite=True, **kwds):
     "dice",
     "hamming",
     "jaccard",
-    "kulsinski",
+    "kulsinski" if sp_version < parse_version("1.8") else "kulczynski1",
```

Comment:
Isn't it rather this?
```suggestion
    (
        *("kulsinski", "kulczynski1")
        if sp_version < parse_version("1.8")
        else "kulczynski1"
    ),
```

This suggestion also applies in other places.

Reply:
"kulczynski1" was introduced in 1.8. Eventually, we could have a transition for 1.8 and 1.9 where both metrics are available which is closer to the deprecation cycle.

Reply:
It would be better since both metrics are not leading to the same results which is something I did not expect at first.